flink/src/main/java/org/apache/iceberg/flink/IcebergTableSource.java

```diff
@@ -112,6 +120,10 @@ public String explainSource() {
       explain += String.format(", LimitPushDown : %d", limit);
     }
 
+    if (isFilterPushedDown()) {
+      explain += String.format(", FilterPushDown,the filters :%s", Joiner.on(",").join(filters));
```

Comment:
Nit: typos in this message. There should be a space after `FilterPushDown,` and the space after `filters` should be after the `:`, not before it. The Joiner should be on `", "` to separate the filters, and it should be a private static final constant.